proposed/http-middleware/middleware-meta.md

```diff
@@ -222,8 +222,8 @@ and a delegate and must return a response. The middleware may:
 - Evolve the request before passing it to the delegate to execute the next
   available middleware.
 - Evolve the response received from the delegate before returning it.
-- Create and return a response without passing it to the delegate, thereby
-  preventing any further middleware from processing.
+- Create and return a response without further process delegation, thereby
+  preventing any inner middleware from processing.
```

Comment:
Granted, it can be seen as an _inner_ middleware.

> How delegates process a request is entirely up to the library and/or individual developer.

Unfortunately, that is not true: 

```php
```